node/impl/full/eth_utils.go

```diff
@@ -731,6 +731,24 @@ func newEthTxReceipt(ctx context.Context, tx ethtypes.EthTx, lookup *api.MsgLook
 		receipt.ContractAddress = &addr
 	}
 
+	var events []types.Event
+	if rct := lookup.Receipt; rct.EventsRoot != nil {
+		events, err = ca.ChainGetEvents(ctx, *rct.EventsRoot)
+		if err != nil {
```

Comment:
should we match a specific error here, or is just any error correct enough ?

Reply:
Worst case, we'll waste some CPU if the datastore is corrupted. Then we'll return the same error below when we try to fetch the events again. Unfortunately, matching errors is kind of error prone.